drivers/ovsSwitch.go

```diff
@@ -400,6 +390,16 @@ func (sw *OvsSwitch) DeletePort(epOper *OvsOperEndpointState) error {
 		return err
```

Comment:
I think this return needs to change to an error log in order to let the clean up continue.